swift-sdk/ITBConsts.swift

```diff
@@ -32,6 +32,20 @@ import Foundation
         public static let inputPlaceholder = ITBL_BUTTON_INPUT_PLACEHOLDER
         public static let action = ITBL_BUTTON_ACTION
     }
+    
+    public class Device {
+        public static let localizedModel = "localizedModel"
+        public static let identifierForVendor = "identifierForVendor"
+        public static let model = "model"
+        public static let systemName = "systemName"
+        public static let systemVersion = "systemVersion"
+        public static let userInterfaceIdiom = "userInterfaceIdiom"
+        public static let deviceId = "deviceId"
+        public static let appPackageName = "appPackageName"
+        public static let appVersion = "appVersion"
+        public static let appBuild = "appBuild"
+        public static let sdkVersion = "sdkVersion"
```

Comment:
Sorry the original ticket description/spec was incorrect, turns out there is a field called `sdkVersion`, and it's used for Android platform version (API level). I've renamed our field to iterableSdkVersion in the Android implementation.

Reply:
Ok so rename sdkVersion => iterableSdkVersion?

Reply:
Yup